lib/auth/accountrecovery.go

```diff
@@ -112,6 +112,8 @@ func (a *Server) verifyCodeWithRecoveryLock(ctx context.Context, username string
 	case err != nil:
 		log.Error(trace.DebugReport(err))
 		return trace.AccessDenied(startRecoveryGenericErrMsg)
```

Comment:
We got a conflict here, as branches v14 and v13 actually look much closer to the modified code than master did.